backend/gateway/gateway.go

```diff
@@ -75,11 +78,7 @@ func RunWithConfig(f *Flags, cfg *gatewayv1.Config, cf *ComponentFactory, assets
 	if err != nil {
 		newTmpLogger().Fatal("could not instantiate logger", zap.Error(err))
 	}
-	defer func() {
-		if err := logger.Sync(); err != nil {
-			panic(err)
-		}
-	}()
+	defer logger.Sync()
```

Comment:
How are you handling the error case here now that it's gone?

Reply:
see https://github.com/uber-go/zap/issues/880